docs/sources/administration/cli.md

```diff
@@ -10,41 +10,217 @@ weight = 8
+### Override default plugin directory
 
+`--pluginsDir value` overrides the path to where your local Grafana instance stores plugins. Use this option if you want to install, update, or remove a plugin somewhere other than the default directory ("/var/lib/grafana/plugins") [$GF_PLUGIN_DIR].
+
+**Example:**
 ```bash
-curl -X PUT -H "Content-Type: application/json" -d '{
-  "oldPassword": "admin",
-  "newPassword": "newpass",
-  "confirmNew": "newpass"
-}' http://admin:admin@<your_grafana_host>:3000/api/user/password
+grafana-cli --pluginsDir "/var/lib/grafana/devplugins" plugins install <plugin-id>
 ```
+
+### Override default plugin repo URL
+
+`--repo value` allows you to download and install or update plugins from a repository other than the default Grafana repo.
+
+**Example:**
+```bash
+grafana-cli --repo "https://example.com/plugins" plugins install <plugin-id>
+```
+
+### Override default plugin .zip URL
+
+`--pluginUrl value` allows you to download a .zip file containing a plugin from a local URL instead of downloading it from the default Grafana source.
+
+**Example:**
+```bash
+grafana-cli --pluginUrl https://company.com/grafana/plugins/<plugin-id>-<plugin-version>.zip plugins install <plugin-id>
+```
+
+### Override Transport Layer Security
+
+`--insecure` allows you to turn off Transport Layer Security (TLS) verification (insecure). You might want to do this if you are downloading a plugin from a non-default source.
+
+**Example:**
+```bash
+grafana-cli --insecure --pluginUrl https://company.com/grafana/plugins/<plugin-id>-<plugin-version>.zip plugins install <plugin-id>
+```
+
+### Enable debug logging
+
+`--debug` or `-d` enables debug logging.
+> Where does is the log stored? Why would you want to use this? Is my example in any way realistic?      
+
+**Example:**
+```bash
+grafana-cli --debug plugins install <plugin-id>
+```
+
+### --configOverrides value  
+> In plain terms, what does this mean?
+Configuration options to override defaults as a string. For example: cfg:default.paths.log=/dev/null
+
+**Example:**
+```bash
+grafana-cli --configOverrides cfg:default.paths.log=/dev/null plugins install <plugin-id>
+```
+
+### --homepath value         
+> In plain terms, what does this mean?
+Sets the path for the Grafana install/home path, defaults to working directory.
+> Why would you want to use this? Is my example in any way realistic?  
+**Example:**
+```bash
+grafana-cli --homepath value plugins install <plugin-id>
+```
+
+### Override config file         
+
+`--config value` overrides the default location where Grafana expects the configuration file. Refer to [Configuration]({{< relref "configuration.md" >}}) for more information about configuring Grafana and default configuration file locations.
+> Why would you want to use this? Is my example in any way realistic?
```

Comment:
Mostly the same as the comment about `homepath`. If the config file is not in a standard location relative to where Grafana is located then this flag might be needed too.

Reply:
Updated. Please check my syntax.